src/index.js

```diff
@@ -10,23 +11,21 @@ const T = new Twit({
   access_token_secret: process.env.ACCESS_TOKEN_SECRET,
 });
 
-/*
+
 setInterval(() => {
   sendRandomImage(T).catch((e) => console.log(e));
-}, 60000);
-*/
+}, 60000 * 5);
```

Comment:
ah ok, might be cool to have this in the package.json? Like botconfig: { postFrequency: 300000} for example

Reply:
here will be env variable

Reply:
or options.file where I can write (if process.env.RETRY_DELAY || 60000 * 5)...

Reply:
process.env is cool as well